logentries/utils.py

```diff
@@ -178,18 +173,20 @@ def flush(self):
     def emit(self, record):
         # Reset stdout. See: http://docs.python.org/2/library/sys.html#sys.__stdout__
         sys.stdout = sys.__stdout__
-        lock.acquire()
-        try:
-            if not self._started and self.good_config:
-                dbg("Starting Logentries Asynchronous Socket Appender")
+        if self.good_config and not self._thread.is_alive():
+            try:
                 self._thread.start()
+            except RuntimeError: # It's already started.
```

Comment:
Hi @graingert. I agreen with your arguments. But, this change prevent the error, `RuntimeError Exception threads can only be started once`? 
This error will now be handled by the logger of django for example?